src/main/java/frc/robot/AngleShooterUtil.java

```diff
@@ -5,20 +5,20 @@
 import edu.wpi.first.math.MathUtil;
 import edu.wpi.first.math.controller.PIDController;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
-import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.subsystems.ElevatorSubsystem;
 
 public class AngleShooterUtil {
     
-    private final ShooterSubsystem s_ShooterSubsystem;
+    private final ElevatorSubsystem s_ElevatorSubsystem;
 
-    private double m_encoderDiff;
+    private static double m_encoderDiff;
```

Comment:
Is there a reason for making this static? If not, we might remove static from this and the updateTargetDiff method.

Reply:
fixed before

Reply:
fixed